aws/resource_aws_ec2_transit_gateway_vpc_attachment.go

```diff
@@ -192,8 +224,10 @@ func resourceAwsEc2TransitGatewayVpcAttachmentRead(d *schema.ResourceData, meta
 		return fmt.Errorf("error setting tags: %s", err)
 	}
 
-	d.Set("transit_gateway_default_route_table_association", (transitGatewayDefaultRouteTableAssociation != nil))
-	d.Set("transit_gateway_default_route_table_propagation", (transitGatewayDefaultRouteTablePropagation != nil))
+	if *transitGateway.OwnerId == vpcOwnerID {
```

Comment:
This conditional should be removed to match with the above change. 👍 Without calling `d.Set()` for the below attributes in the RAM shared Transit Gateway scenario, operators will be presented a plan immediately after import similar to:

```
"transit_gateway_default_route_table_association": "" => "true"
"transit_gateway_default_route_table_propagation": "" => "true"
```